package.json

```diff
@@ -34,9 +34,10 @@
   },
   "homepage": "https://github.com/numo-labs/lambda-ne-dynamic-package-provider#readme",
   "dependencies": {
-    "async": "2.0.0-rc.5",
     "aws-lambda-helper": "^2.18.0",
+    "bluebird": "^3.4.1",
```

Comment:
is it worth it to include a promise library? 
can native 4.3.2 node version do the same without adding a dependency?